src/StoreReviewTest/StoreReviewTest/MainPage.xaml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Plugin.StoreReview;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace StoreReviewTest
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+		async void Button_Clicked(object sender, EventArgs e)
+		{
+			await CrossStoreReview.Current.RequestReview(true);
```

Comment:
The PR looks good to me. @jamesmontemagno 
Should we add a pre-compile check here so users know to set it to false for release?
```
bool isTestMode = false;
#if DEBUG
isTestMode = true;
#endif 
await CrossStoreReview.Current.RequestReview(isTestMode);
```
Also, let me know if you would like me to test it too